tap_github/tests/test_authenticator.py

```diff
@@ -112,3 +119,308 @@ def test_has_calls_remaining_fails_if_few_calls_remaining_and_reset_time_not_rea
         token_manager.update_rate_limit(mock_response_headers)
 
         assert not token_manager.has_calls_remaining()
+
+
+class TestAppTokenManager:
+
+    def test_initialization_with_3_part_env_key(self):
+        with patch.object(AppTokenManager, "claim_token", return_value=None):
+            token_manager = AppTokenManager("12345;;key\\ncontent;;67890")
+            assert token_manager.github_app_id == "12345"
+            assert token_manager.github_private_key == "key\ncontent"
+            assert token_manager.github_installation_id == "67890"
+
+    def test_initialization_with_2_part_env_key(self):
+        with patch.object(AppTokenManager, "claim_token", return_value=None):
+            token_manager = AppTokenManager("12345;;key\\ncontent")
+            assert token_manager.github_app_id == "12345"
+            assert token_manager.github_private_key == "key\ncontent"
+            assert token_manager.github_installation_id == ""
```

Comment:
I think that https://github.com/MeltanoLabs/tap-github/blob/aade338aaa5fe7a49427e3b7596f9ed45a3bae87/tap_github/authenticator.py#L132 this line is expecting this to be None not `""`

I think we either want to parse this as empty string and then change the other code to look for "falsy" or we want to parse this as "None" and then we do not need to change the code.

I have been getting
```
404 Client Error: Not Found for url: https://api.github.com/app/installations//access_tokens
```
post this change

Reply:
Hey @camerondavison, thanks for reporting and sorry this change caused an issue.  Taking a look now. 👀

Reply:
Put up a PR to fix this, if you like you can try pegging your version of the tap to the commit `38e1bdf9381c3a3fa17cceada9e1ede7be57ae6b` to see if it fixes your issue.

Reply:
I've also tagged v1.4.8